### DISCUSSION TITLE
Changing user role

### DISCUSSION THREAD

**Discussion by Dandrizzy** [supabase/supabase#22613] [Questions]
Please how do I change a user's role in supabase please this would save me a lot of time and trouble thanks

**Comment by Nunyaklah** [comment]
Hi this documentation might be of help , this is on Role based access control: https://supabase.com/docs/guides/auth/custom-claims-and-role-based-access-control-rbac and this is on row level security: https://supabase.com/docs/guides/auth/row-level-security . I hope they are of help
